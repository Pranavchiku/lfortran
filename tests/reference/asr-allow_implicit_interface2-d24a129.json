{
    "basename": "asr-allow_implicit_interface2-d24a129",
    "cmd": "lfortran --indent --show-asr --implicit-typing --implicit-interface --no-color {infile} -o {outfile}",
    "infile": "tests/allow_implicit_interface2.f90",
    "infile_hash": "1f457aff9cece7a1bae818adeed2dfdb5998c30ee4ff92f4cb11c692",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-allow_implicit_interface2-d24a129.stdout",
    "stdout_hash": "f70f37a50a63c6c026038e5d3c1f3f9b0476e08351266702e10b423b",
    "stderr": "asr-allow_implicit_interface2-d24a129.stderr",
    "stderr_hash": "e442416a1b51fa0313024ebd5d2769b7c8d3eed237fb13090c228ece",
    "returncode": 0
}