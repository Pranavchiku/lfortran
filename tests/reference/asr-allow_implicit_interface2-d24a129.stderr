warning: Implied save feature
 --> tests/allow_implicit_interface2.f90:2:1
  |
2 | integer :: n, x, y
  | ^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/allow_implicit_interface2.f90:2:1
  |
2 | integer :: n, x, y
  | ^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/allow_implicit_interface2.f90:2:1
  |
2 | integer :: n, x, y
  | ^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
